### PR TITLE
ref(tests): fix flaky test_organization_metric_data.py tests [TET-378]

### PR DESCRIPTION
**Pull request by andriisoldatenko**
After deep debugging with @ahmedetefy i modified a test:
```diff
diff --git tests/sentry/api/endpoints/test_organization_metric_data.py tests/sentry/api/endpoints/test_organization_metric_data.py



@@ -1169,17 +1169,22 @@ class OrganizationMetricDataTest(MetricsAPIBaseTestCase):

         assert len(response.data["groups"]) == 1

     def test_one_field_orderby_with_no_groupby_returns_one_row(self):
         # Create time series [1, 2, 3, 4] for every release:
+        print()
         for minute in range(4):
             for _ in range(minute + 1):
                 # One for each release
                 for release in ("foo", "bar", "baz"):
+                    started = (time.time() // 60 - 3 + minute) * 60
+                    print(f"started = {started} for release {release}")
+                    time.sleep(1)
                     self.store_session(
                         self.build_session(
                             project_id=self.project.id,
-                            started=(time.time() // 60 - 3 + minute) * 60,
+                            started=started,
                             release=release,
                         )
                     )
```
i was able to reproduce locally issue:

```
――――――――――――――――――――――――――――――――――――― OrganizationMetricDataTest.test_one_field_orderby_with_no_groupby_returns_one_row ―――――――――――――――――――――――――――――――――――――
tests/sentry/api/endpoints/test_organization_metric_data.py:1204: in test_one_field_orderby_with_no_groupby_returns_one_row
    assert group["series"]["sum(sentry.sessions.session)"] == [3, 6, 9, 12]
E   AssertionError: assert [6.0, 9.0, 6.0, 6.0] == [3, 6, 9, 12]
E     At index 0 diff: 6.0 != 3
E     Use -v to get the full diff

 tests/sentry/api/endpoints/test_organization_metric_data.py ⨯     
```

each time when i run a test and if we add `freeze_time`:
```
-    @freeze_time((datetime.now() - timedelta(hours=1)).replace(minute=30))
+    # @freeze_time("2022-09-13 04:10:00", tick=True)
+    # Sep 20, 2022 10:00:02 AM UTC
     def test_one_field_orderby_with_no_groupby_returns_one_row(self):
         # Create time series [1, 2, 3, 4] for every release:
+        print()
         for minute in range(4):
             for _ in range(minute + 1):
                 # One for each release
                 for release in ("foo", "bar", "baz"):
+                    started = (time.time() // 60 - 3 + minute) * 60
+                    print(f"started = {started} for release {release}")
+                    time.sleep(1)
                     self.store_session(
                         self.build_session(
                             project_id=self.project.id,
-                            started=(time.time() // 60 - 3 + minute) * 60,
+                            started=started,
                             release=release,
                         )
                     )
```
we can see it works 🎉

### CODE REVIEW

File: tests/sentry/api/endpoints/test_organization_metric_data.py [hunk 1]
```diff
@@ -37,6 +37,7 @@ def indexer_record(use_case_id: UseCaseKey, org_id: int, string: str) -> int:
 
 
 @region_silo_test
+@freeze_time()
```
Comment (asottile-sentry) [inline tests/sentry/api/endpoints/test_organization_metric_data.py:40]: alternatively -- does it also fix it if time is only retrieved once?  I assume the flakiness is due to a second ticking over in the loop?

**Review by asottile-sentry — APPROVED** [review]

File: tests/sentry/api/endpoints/test_organization_metric_data.py (hunk 1 above)
Comment (andriisoldatenko) [reply tests/sentry/api/endpoints/test_organization_metric_data.py:40]: do you mean if we move `now = time.time()` out of the all loops and rewrite it `started = (now // 60 - 3 + minute) * 60`?

Comment (asottile-sentry) [reply tests/sentry/api/endpoints/test_organization_metric_data.py:40]: I think so?  I'm not 100% that it will fix it though

**Pull request merged**